src/test/java/seedu/guest/model/guest/RoomTest.java

```diff
@@ -25,20 +25,24 @@ public void isValidRoom() {
         assertThrows(NullPointerException.class, () -> Room.isValidRoom(null));
 
         // invalid room
-        assertFalse(Room.isValidRoom("")); // empty string
-        assertFalse(Room.isValidRoom(" ")); // spaces only
+        assertFalse(Room.isValidRoom("")); // empty string, near boundary values
+        assertFalse(Room.isValidRoom(" ")); // spaces only, near boundary values
         assertFalse(Room.isValidRoom("-50-23")); // not allowed to start with a hyphen
         assertFalse(Room.isValidRoom("50 - 23")); // not allowed to have spacings in between
```

Comment:
Can we change this to only 1 space to be closer to the boundary of "between 2 alphanumeric characters"?

Reply:
Agreed!